carlos/rocker/css/style.css

```diff
@@ -0,0 +1,272 @@
+html {
+    box-sizing: border-box;
+    font-size: 62.5%;
+    /** 1rem = 10px **/
+}
+
+*,
+*:before,
+*:after {
+    box-sizing: inherit;
+}
+
+body {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 1.6rem;
+    line-height: 2;
+}
+
+
+/** Globales **/
+
+.contenedor {
+    max-width: 1200px;
+    width: 95%;
+    margin: 0 auto;
+}
+
+h1,
+h2,
+h3,
+h4 {
+    font-family: 'PT Sans', sans-serif;
+}
+
+h1 {
+    font-size: 4.8rem;
+}
+
+h2 {
+    font-size: 4rem;
+    line-height: 1.2;
+}
+
+h3 {
+    font-size: 3.2rem;
+}
+
+h4 {
+    font-size: 2.8rem;
+}
+
+img {
+    max-width: 100%;
+}
+
+
+/** Utilidades **/
+
+.centrar-texto {
+    text-align: center;
+}
+
+.no-margin {
```

Comment:
This style block naming pattern was popularized by libraries such as material ui and others, when using BEM there is no need to have "special classes" for things like this, you force other devs to look for what is the way you are achieving the result the class name advertises. Also, centering text as in your style rule `centrar-texto`, "text-align" is not the only way to do it, if you had a flex container you could have used "justify content: center". Since there are more than a single way to implement a style that is why you want to avoid using reusable classes for things like centering.